### PR TITLE
docs: write "Step 3: Perform fit"

### LINKED ISSUE #68: Write notebook about how to perform a fit
Can be done after #73 

This notebook should reflect [Step 3 of the ComPWA Workflow](https://compwa.github.io/usage/workflow/3_perform_fit.html). The notebook can be placed under [`examples/jupyter`](https://github.com/ComPWA/tensorwaves/tree/master/examples/jupyter).

_Originally posted by @redeboer in https://github.com/ComPWA/tensorwaves/pull/65_

### PR DESCRIPTION
**Pull request by redeboer**
Closes #68 

This PR resulted in #99

### CODE REVIEW

**Comment by redeboer** [comment]
https://travis-ci.com/github/ComPWA/tensorwaves/jobs/353834448
Not sure what's going on here, but for sure the amplitude model is too complex (really?) to be run in a notebook. Also relates to #100. Maybe for now creating a simpler model in Step 1 is a solution?

File: examples/workflow/3_perform_fit.ipynb [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from copy import deepcopy\n",
+    "\n",
+    "initial_parameters = deepcopy(estimator.parameters)\n",
```
Comment (spflueger) [inline examples/workflow/3_perform_fit.ipynb:120]: why you make a deepcopy here?

File: examples/workflow/2_generate_data.ipynb [hunk 2]
```diff
@@ -278,13 +278,29 @@
...
+   ]
+  },
+  {
+   "cell_type": "raw",
    "metadata": {
     "raw_mimetype": "text/restructuredtext"
    },
    "source": [
-    "We first decrease the debugging level a bit to avoid some redundant warnings."
+    "`tensorwaves` currently has no export functionality for data samples. However, as we work with `numpy.ndarray`s, it's easy to just `'pickle' <https://docs.python.org/3/library/pickle.html>`_ these data samples with `numpy.save`:"
```
Comment (spflueger) [inline examples/workflow/2_generate_data.ipynb:303]: Here the links also render incorrectly

File: examples/workflow/3_perform_fit.ipynb (hunk 1 above)
Comment (redeboer) [reply examples/workflow/3_perform_fit.ipynb:120]: Oh oops yes that was a debugging attempt

File: examples/workflow/3_perform_fit.ipynb [hunk 3]
```diff
@@ -115,10 +115,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from copy import deepcopy\n",
-    "\n",
-    "initial_parameters = deepcopy(estimator.parameters)\n",
-    "initial_parameters"
+    "estimator.parameters"
```
Comment (spflueger) [inline examples/workflow/3_perform_fit.ipynb:118]: i meant like `initial_parameters = estimator.parameters`
and then just following the code

**Pull request merged**